content/docs/node-operators/install-aut/_index.md

```diff
@@ -148,6 +123,14 @@ sudo systemctl restart docker
    For more information on using and pulling Docker images from GHCR, see GitHub docs [Working with the container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry).
    {{< /alert >}}
 
+1. Verify the authenticity of the Autonity Docker images against the official [image digests](https://github.com/autonity/autonity/pkgs/container/autonity/versions):
```

Comment:
Shouldn't this one be point number 2?

Reply:
Markdown automatically numbers them sequentially, if each is simply "1".

See:

1. first item
1. second item
1. third item

Look at the source code of this, and see that I actually started each line with "1"

Reply:
😮 didn't know about this one

Reply:
Ok, resolved then!